lib/blockchain/cardano.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path')
+const common = require('./common')
+const mnemonicHelpers = require('../../lib/mnemonic-helpers')
+
+const coinUtils = require('../coin-utils')
+const options = require('../options')
+const _ = require('lodash')
+
+module.exports = { setup }
+
+const es = common.es
+
+function setup(dataDir) {
+  const coinSettings = coinUtils.getCryptoCurrency('ADA')
+
+  // original Cardano node script suppose to have DB and other things in
+  // in `state-wallet-testnet` folder (relative to him), but we wanna to use
+  // Lamassu recommended path defined in `dataDir` variable
+  es(
+    `sudo sed -i 's#state-wallet-testnet/#${_.escapeRegExp(dataDir).replace(
+      /#/g,
+      '\\#'
+    )}/#g' /usr/local/bin/cardano-node`
+  )
+
+  common.firewall([coinSettings.defaultPort])
+
+  const defaultConfigData = {
```

Comment:
maybe add a comment why init data is empty (is empty because walletId & index will be fetched after first call API, now we are not sure that cardano node running)

Reply:
also maybe we can use functions from `lib/plugins/wallet/cardano/wallet.js` to have this (saving configuration) only on one place